rbac/internal/views.py

```diff
@@ -521,12 +521,13 @@ def bootstrap_tenant(request):
     logger.info("Running bootstrap tenant.")
 
     org_id = request.GET.get("org_id")
+    force = bool(request.GET.get("force", "false"))
```

Comment:
can we put this param  into[ internal api spec](https://github.com/RedHatInsights/insights-rbac/blob/master/rbac/internal/specs/openapi.json) ?